nhost/compose/config.go

```diff
@@ -171,6 +171,27 @@ func (c Config) postgresConnectionString() string {
 	return fmt.Sprintf("postgres://%s:%s@%s:%d/%s", user, password, SvcPostgres, dbPort, db)
 }
 
+func (c Config) hasuraConnectionString() string {
```

Comment:
Why is this called a hasura connection string and not postgres?

Reply:
You could just change the code in `postgresConnectionString()`

Reply:
Because this is the connection string we pass to hasura. I am changing it to 1 function only.